.github/workflows/stale.yml

```
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 19 * * 2'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open for more than 14 days with no activity. Remove stale label or comment or this will be closed in 14 days.'
+          stale-pr-message: 'This PR is stale because it has been open for more than 14 days with no activity. Remove stale label or comment or this will be closed in 14 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 28 days with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 28 days with no activity.'
```

Purely a suggestion since I'm fond of YAML and hate long lines. But we can utilize some of my favorite features, such as multiline folding and stripping (`>-`):

```suggestion
          stale-issue-message: >-
            This issue is stale because it has been open for more than 14 days with no activity. 
            Remove stale label or comment or this will be closed in 14 days.
          stale-pr-message: >-
            This PR is stale because it has been open for more than 14 days with no activity.
            Remove stale label or comment or this will be closed in 14 days.
          close-issue-message: >-
            This issue was closed because it has been stalled for 28 days with no activity.
          close-pr-message: >- 
            This PR was closed because it has been stalled for 28 days with no activity.
```

---

It irks me that these lines are wrapped in this preview. 😆